Items/Weapons/UndyingSpear.cs

```diff
@@ -40,9 +42,28 @@ public override void SetDefaults()
 			Item.noUseGraphic = true; //No swing animation
 			Item.DamageType = DamageClass.Magic;
 			Item.crit = 10;
-			Item.glowMask = MiscGlowMasks.UndyingSpear;
 		}
-		
+
+		public override void PostDrawInInventory(SpriteBatch spriteBatch, Vector2 position, Rectangle frame, Color drawColor, Color itemColor, Vector2 origin, float scale)
```

Comment:
If you really have to use custom drawing code for every glowmask item, then it's probably better to move all the repeated code elsewhere. Maybe a subclass of ModItem like GlowmaskModItem that has its own Texture2D modGlowMask field.

Reply:
Done. I think I found a better name for it. Also the issue was that the old method conflicted with another mod was doing the same thing so it expected the size to be the same since the load. The other mod changed the size causing out of bounds exception on unload. I probably could have fixed it up but this will never have conflicts with reasonable code.